### ISSUE TITLE
Uncaught Error: The browser threw an error while playing the video {URL}: Code 2 - PIPELINE_ERROR_READ: FFmpegDemuxer: data source error.

### ISSUE THREAD

**Issue by elliottamador** [remotion-dev/remotion#2367]
# Bug Report 🐛

I am using a `<Player>` in my webapp with `<Video>` component's that use a Cloudfront MP4 url for the src. The Player starts out fine, I can watch the video and seek around to different parts but eventually this error is thrown. Monitoring the Network tab, I do not see any of the requests to my video in Cloudfront failing, they all succeed with valid Content-Range headers and a 206 response code. To work around the issue, I've had to add an `onError` handler that appends the current timestamp to the video URL as a query string to essentially invalidate the cached video response. 

Here is some relevant code:

```
return (
  <Video
    src={videoUrl}
    startFrom={getDurationInFps(video.start, fps)}
    endAt={getDurationInFps(video.end, fps)}
    style={videoStyle}
    acceptableTimeShiftInSeconds={ACCEPTABLE_TIME_SHIFT_SECONDS}
    onError={(error) => {
	    console.log("error in Video: ", error);
	    setVideoUrl(video.src_url + "?time=" + Date.now());
    }}
  />
);
```

```
<Player
  ref={videoPlayerRef}
  component={VideoComposition}
  inputProps={{
	  videos,
	  fps,
	  width,
	  height,
	  aspectRatio,
	  subtitlesEnabled,
	  subtitleSettings,
	  videoTranscriptions,
	  showCropping: true,
  }}
  durationInFrames={durationInFrames}
  compositionWidth={width}
  compositionHeight={height}
  fps={fps}
  clickToPlay={true}
  inFrame={inFrame}
  outFrame={outFrame}
  showPosterWhenPaused
  renderPoster={renderPoster}
  spaceKeyToPlayOrPause={true}
  moveToBeginningWhenEnded={true}
/>
```

**Comment by JonnyBurger** [comment]
Hey @elliottamador!

This is unfortunate, but sadly completely out of our control. Remotion is using a regular `<video>` tag and seeking around based on the Player position. 
It seem like this is enough to throw Chrome off and throw this error.

During rendering, I've implemented a fix for 4.0 so that we don't inherit browser bugs. For more details, I made a video about these types of bugs: https://twitter.com/JNYBGR/status/1650078296523022337

During Preview in the browser, we unfortunately inherit all browser bugs and I think you are already using the best solution for working around it.
